### ISSUE TITLE
[Feature]: Compatibility with Traveller's Backpacks and Tom's Simple Storage Mod

### ISSUE THREAD

**Issue by BunCat09** [TheIllusiveC4/Polymorph#267]
### What is the new feature or improvement?

Currently, the 1.20.4 version of the mod won't show the button for Traveller's Backpacks' Crafting Upgrade and Tom's Simple Storage Mod's Crafting Terminal. It would be awesome if you could make the mod compatible with them in the future.

**Comment by uczpre** [comment]
That would be really cool !

**Comment by ffuentesm** [comment]
Yeah, Tom's Storage stopped working with Polymorth in Neoforge 1.21.1
toms_storage-1.21-2.0.9

**Comment by Soforak** [comment]
I'd like this feature to be implemented in 1.21.1 please!

**Comment by SirJeffersonIII** [comment]
Seconded for 1.21.1. Makes tom's not as easy to use in BMC3

**Comment by sprocketaudio** [comment]
I would also like Polymorph to work with Toms for 1.21.1 please!
